portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -0,0 +1,47 @@
+const { startOfMonth, format, subMonths } = require('date-fns');
+const externalApi = require('../../../external-api/api');
```

Comment:
I've been thinking and I think I prefer having notify logic in external API for this as we don't care about response times etc., though given on Francesca's work it brings in a delay of another HTTPS call i'm a little more apprehensive. Let's leave it like this but discuss it next week if we reckon this is neater and worth the small performance hit on portal routes

Reply:
It feels like `external-api` should just be a shared package that provides an interface for external providers. That way we'd have the best of both worlds - no extra API requests, and shared code across all of the dtfs2 packages.